app/js/main.js

```diff
@@ -725,9 +758,11 @@ function _writeFile(file, data) {
     "use strict";
      fs.writeFile(file, data, function(err) {
         if (err) {
-            throw err;
+            console.error(`Failed to save ${file} to disk`);
```

Comment:
Again, add `console.error(err);` before calling `createSaveDirectory()` so we can get details.